govcd/catalog.go

```diff
@@ -61,34 +61,45 @@ func NewAdminCatalog(client *Client) *AdminCatalog {
 
 // Deletes the Catalog, returning an error if the vCD call fails.
 // Link to API call: https://code.vmware.com/apis/220/vcloud#/doc/doc/operations/DELETE-Catalog.html
-func (adminCatalog *AdminCatalog) Delete(force, recursive bool) error {
-	adminCatalogHREF := adminCatalog.client.VCDHREF
-	adminCatalogHREF.Path += "/admin/catalog/" + adminCatalog.AdminCatalog.ID[19:]
+func (catalog *Catalog) Delete(force, recursive bool) error {
+	adminCatalogHREF := catalog.client.VCDHREF
+	adminCatalogHREF.Path += "/admin/catalog/" + catalog.Catalog.ID[19:]
```

Comment:
Please add a comment what "19" means.

Reply:
I am not sure, not mine code. Need to debug to figure out.

Reply:
Changed code to be concise with rest of the code.